Modix.Services/Tags/TagInlineParserBehavior.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Modix.Data.Models.Core;
+using Modix.Services.Core;
+
+namespace Modix.Services.Tags
+{
+    public class TagInlineParserBehavior : BehaviorBase
```

Comment:
Genuine question, what's the advantage of making this a behavior vs making it an `INotificationHandler<ChatMessageReceived>`?

Reply:
Nothing at all. Fixed!